test/e2e/case2_config_deployment_test.go

```diff
@@ -376,6 +414,7 @@ var _ = Describe("Test config-policy-controller deployment", Ordered, func() {
 						g.Expect(args).To(ContainElement("--evaluation-concurrency=5"))
 						g.Expect(args).To(ContainElement("--client-max-qps=50"))
 						g.Expect(args).To(ContainElement("--leader-elect=false"))
+						g.Expect(args).To(ContainElement("--enable-operator-policy=true"))
```

Comment:
This is looking good--thanks for adding some tests! Would you update the title of this step to replace the "new logging level and concurrency" with something more generic?